src/overlays/actors/ovl_En_Snowman/z_en_snowman.c

```diff
@@ -642,14 +642,14 @@ void EnSnowman_Melt(EnSnowman* this, GlobalContext* globalCtx) {
         }
     }
 
-    this->actor.scale.y = this->timer * 0.0002f * this->eenoScale;
-    this->actor.scale.x = (this->eenoScale * 0.0139999995f) - (0.4f * this->actor.scale.y);
-    this->actor.scale.z = (this->eenoScale * 0.0139999995f) - (0.4f * this->actor.scale.y);
+    this->actor.scale.y = this->work.timer * 0.0002f * this->eenoScale;
+    this->actor.scale.x = (this->eenoScale * (1.4f / 100.0f)) - (0.4f * this->actor.scale.y);
+    this->actor.scale.z = (this->eenoScale * (1.4f / 100.0f)) - (0.4f * this->actor.scale.y);
```

Comment:
I suspect that 0.01f is going to me more in line with the rest of these eventually, but it doesn't really matter.

Reply:
Yeah, good point. I'll switch it to 0.01f.